Don't import VRDL trees

ttung opened the pull request:
### Description
They don't exist!

Depends on #370 

#### Issue
[ch136615](https://app.clubhouse.io/genepi/story/136615/vrdl-trees-do-not-port)

### Test plan
removed all but vrdl and tulare and ran import.

shanaxel42 commented on scripts/migrate-all.sh:
```diff
@@ -38,21 +38,23 @@ done
 for county in "${COUNTIES[@]}"; do
-    aspen_group_id=$(echo "$COUNTY_INFO" | jq -r ".$county".aspen_group_id)
-    aspen-cli db --local import-covidhub-trees --covidhub-aws-profile biohub --s3-src-prefix s3://covidtracker-datasets/cdph/"$county" --s3-key-prefix /imported/phylo_trees/"$county" --aspen-group-id "$aspen_group_id"
+    if [ "$(echo "$COUNTY_INFO" | jq ".$county.skip_tree_import")" != "true" ]; then
```
wait where does `$county.skip_tree_import` come from?

phoenixAja replied:
i was wondering the same thing (also about `county.aspen_group_id`), some comments would be great in this script for those of us who arn't shell experts

ttung replied:
> wait where does $county.skip_tree_import come from?

It's set for vrdl.

ttung replied:
oh wait, i missed that in the checkin. #fml

ttung replied:
> i was wondering the same thing (also about `county.aspen_group_id`), some comments would be great in this script for those of us who arn't shell experts 

Added docs in #379

Commit:
Don't import VRDL trees

### Description
They don't exist!

#### Issue
[ch136615](https://app.clubhouse.io/genepi/story/136615/vrdl-trees-do-not-port)

### Test plan
removed all but vrdl and tulare and ran import.

shanaxel42 reviewed (approved):
nice!
